conf/janus.eventhandler.mqttevh.jcfg.sample

```diff
@@ -46,4 +47,7 @@ general: {
 	#tls_client_key = "/path/to/key.pem"
 	#tls_ciphers
 	#tls_version
+
+	# These options work with MQTT 5 only.
+	#add_user_properties = ()	# List of user property ["key", "value"] pairs to add.
```

Comment:
It's nice to see the list/array features of libconfig finally used somewhere :slightly_smiling_face: